teacher.rb

```diff
@@ -0,0 +1,13 @@
+require './person'
+
+# Class teacher
+class Teacher < Person
+  def initialize(name, age, specialization, parent_permission: true)
```

Comment:
- [ ] The name parameter should have a default value of `Unknown`.

![name](https://user-images.githubusercontent.com/53124548/223476122-52068fe6-0963-4b54-859d-3b0bfd16cd75.png)